basics/indexing/star-tree-index.md

```diff
@@ -1,16 +1,22 @@
+---
+description: Learn what a star-tree index is and how one works.
+---
```

Comment:
to be sure, think were these edits all in the other PR I just reviewed? ...so didn't review here

Reply:
they are. I'll need to merge the others first. I _think_ I know what happened...git quirks I didn't anticipate, but should be able to avoid in the future.